examples/logs-advanced/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "logs-advanced"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+rust-version = "1.75.0"
+publish = false
+autobenches = false
+
+[[bin]]
+name = "logs-advanced"
+path = "src/main.rs"
+bench = false
+required-features = ["spec_unstable_logs_enabled"]
+
+[features]
+spec_unstable_logs_enabled = ["opentelemetry/spec_unstable_logs_enabled", "opentelemetry_sdk/spec_unstable_logs_enabled"]
+
+[dependencies]
+opentelemetry_sdk = { workspace = true, features = ["logs", "spec_unstable_logs_enabled"] }
+opentelemetry-stdout = { workspace = true, features = ["logs"] }
+opentelemetry-appender-tracing = { workspace = true, features = ["spec_unstable_logs_enabled"] }
+tracing = { workspace = true, features = ["std"]}
+tracing-subscriber = { workspace = true, features = ["env-filter","registry", "std", "fmt"] }
+opentelemetry = { version = "0.31.0", features = ["spec_unstable_logs_enabled"] }
```

Comment:
lets do workspace version?

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry-rust/pull/3266/commits/90b9752749a2ac828e8c4a067d434c00d9a7c3fe